src/compile/model.ts

```diff
@@ -238,16 +238,6 @@ export abstract class Model {
       legends: {}
     };
   }
-  private initTitle(title: TitleParams<ExprOrSignalRef>) {
```

Comment:
Can you explain why we can remove this method?

Reply:
It's only used in the constructor and I replaced that call with a call to `replaceExprRef`, which seems to do the same thing.